File: testdata/bench/perl/BenchmarkTest00003.pl
use Mojolicious::Lite;
get '/search' => sub {
    my $c = shift;
    my $query = $c->param('q');
    my $dbh = DBI->connect("dbi:Pg:dbname=app", "", "");
    $dbh->do("SELECT * FROM items WHERE title LIKE '%$query%'");
};
app->start;

File: testdata/bench/lua/BenchmarkTest00003.lua
local pgmoon = require "pgmoon"
local args = ngx.req.get_uri_args()
local id = args.id
local pg = pgmoon.new({host="127.0.0.1", database="app"})
pg:connect()
local query = string.format("SELECT * FROM orders WHERE id = %s", id)
local res = pg:query(query)
ngx.say(res)
